pages/realms/components/RealmsDashboard.tsx

```diff
@@ -20,7 +21,17 @@ export default function RealmsDashboard({
 }) {
   const router = useRouter()
   const { fmtUrlWithCluster } = useQueryContext()
+
   const { connected, current: wallet } = useWalletStore((s) => s)
+  const [counts, setCounts] = useState({})
+
+  useEffect(() => {
+    const fetchBadgeCounts = async () => {
+      setCounts(await getNumberOfProposalsInVotingState(realms))
+    }
+    // xxx: exclude badge counts for uncharted realms for now
```

Comment:
I think that's ok, being charted should come with privileges and lots of spam in the uncharted territory shouldn't slow us down